install/kubernetes/templates/helm/istio/values.yaml.tmpl

```diff
@@ -73,6 +73,7 @@ sidecarInjector:
   #  cpu: 100m
   #  memory: 128Mi
   nodeSelector: {}
+  includeIPRanges: {}
```

Comment:
Docs please: what is the syntax, example (with 2 ranges to see the delim ), what is the effect of setting it up ( disable egress rules )

Reply:
Also - not sure if the naming is right, it may be quite confusing. 
iptablesCaptureWhitelist ?

Reply:
@costinm I added a description there in the values.yaml.tmpl with a link to the istioctl docs. I can change the name if we want but I think it makes sense for it to match the corresponding argument to `istioctl kube-inject`.